src/main/java/org/rutebanken/tiamat/config/Messages.java

```diff
@@ -0,0 +1,44 @@
+package org.rutebanken.tiamat.config;
```

Comment:
I see that the class MessageConfig is configuration. But this class responsibility is to retrieve messages. Should it be moved to pacakage org.rutebanken.tiamat.messages or similar?